docker-compose.M1.yaml

```diff
@@ -101,6 +100,7 @@ services:
       - redis
       - jaeger
       - otelcollector
+      - keycloak
```

Comment:
I think this file also needs the definition added for keycloak like it is in the other file